quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/update/QuarkusCli38to315UpdateIT.java

```diff
@@ -18,11 +18,11 @@
 import io.quarkus.test.util.IQuarkusCLIAppManager;
 
 @Tag("quarkus-cli")
-public class QuarkusCli35to315UpdateIT extends AbstractQuarkusCliUpdateIT {
-    private static final DefaultArtifactVersion oldVersion = new DefaultArtifactVersion("3.5");
+public class QuarkusCli38to315UpdateIT extends AbstractQuarkusCliUpdateIT {
```

Comment:
Afaik don't have problem with rename, but isn't this cause the problem when backporting as this file already exist.